modopt/tests/test_opt.py

```diff
@@ -21,94 +24,147 @@
     import_sklearn = True
 
 
-class dummy(object):
+dummy_func = lambda x_val: x_val
+dummy_func_double = lambda x_val: x_val * 2
+dummy_func_sq = lambda x_val: x_val ** 2
+dummy_func_cube = lambda x_val: x_val ** 3
```

Comment:
Why `dummy_` as a prefix? I assume this is because you wanted to get rid of unwanted pep8 warnings, but I think that it would be preferable to change the case to BIG rather than adding `dummy_`. It makes the reader think that these functions serve no purpose when they actually do.

Reply:
I see it's also because some of them were actually used as dummy. I think it's even more important to make the difference then.

Reply:
Yeah, fair point. I have removed the `dummy_` prefix. `func` on it's own would raise errors but with the suffixes everything is fine. I also added a comment line to explain what these lambda functions are for.